src/main/java/org/folio/circulation/resources/AllowedServicePointsResource.java

```diff
@@ -99,6 +100,13 @@ private static Result<AllowedServicePointsRequest> buildRequest(RoutingContext r
         useStubItem);
       errors.add(String.format("useStubItem is not a valid boolean: %s.", useStubItem));
     }
+    if (ecsRequestRouting != null && !"true".equals(ecsRequestRouting)
```

Comment:
Since we are validating boolean string twice now, do you think this check deserves a separate method? Something like `isIvalidBooleanString(String boolean)`